### PR TITLE
[bugfix] Append code `val linkisVar=1` to prevent bugs that do not exit tasks for a long time

### LINKED ISSUE #4759: [Bug] the task of scala execution cannot be stopped
### Search before asking

- [X] I searched the [issues](https://github.com/apache/linkis/issues) and found no similar issues.

### Linkis Component

linkis-computation-governance, linkis-engineconnn-plugin

### Steps to reproduce

Task execution cannot be stopped when the last line of a scala script is a comment
the code:
`
import org.apache.spark.sql.functions._
import org.apache.spark.sql.expressions._
import org.apache.spark.sql.types._
case class Node(id: String, children: Seq[Node])
// 构造样例数据
val relationships = Seq(("A", "B"), ("B", "C"), ("B", "D"), ("A", "E")).toDF("parent", "child")
println(relationships)
//println("") 
`

b266fa5fcbf341af96e3b877bd4720f

### Expected behavior

The task ends normally.

### Your environment

- Linkis version used: 1.3.2

### Anything else

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by guoshupei**
this close #4759

### What is the purpose of the change

EngineConn-Core defines the the abstractions and interfaces of the EngineConn core functions.
The Engine Service in Linkis 0.x is refactored, EngineConn will handle the engine connection 
and session management.

### Related issues/PRs

Related issues: #4759 

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

**Comment by aiceflower** [comment]
LGTM.

File: linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/interceptor/impl/CommentInterceptor.scala [hunk 1]
```diff
@@ -102,7 +103,15 @@ object ScalaCommentHelper extends CommentHelper {
   override val commentPattern: Regex = """^\s*//.+\s*""".r.unanchored
   private val scalaCommentPattern: String = "(?ms)([\"'](?:|[^'])*['\"])|//.*?$|/\\*.*?\\*/"
 
-  override def dealComment(code: String): String = code
+  override def dealComment(code: String): String = {
+    // Append code `val a=1` to prevent bugs that do not exit tasks for a long time
+    if (EntranceConfiguration.ENTRANCE_SCALA_APPEND_CODE_ENABLED.getValue) {
```
Comment (peacewong) [inline linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/interceptor/impl/CommentInterceptor.scala:108]: Is it more reasonable to modify Scala Code Parser, go to append val linkisVar=1

Comment (guoshupei) [reply linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/interceptor/impl/CommentInterceptor.scala:108]: Yes, Clearer meaning

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**